astronomer/providers/google/cloud/example_dags/example_bigquery_sensors.py

```diff
@@ -0,0 +1,98 @@
+"""
+Example Airflow DAG for Google BigQuery Sensors.
+"""
+import os
+from datetime import datetime
+
+from airflow import models
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryCreateEmptyTableOperator,
+    BigQueryDeleteDatasetOperator,
+    BigQueryInsertJobOperator,
+)
+from airflow.providers.google.cloud.sensors.bigquery import (
+    BigQueryTablePartitionExistenceSensor,
+)
+
+from astronomer.providers.google.cloud.sensors.bigquery import (
+    BigQueryTableExistenceSensorAsync
+)
+
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "astronomer-airflow-providers")
+DATASET_NAME = os.environ.get("GCP_BIGQUERY_DATASET_NAME", "astro_dataset")
+LOCATION = "us"
+
+TABLE_NAME = "partitioned_table"
+INSERT_DATE = datetime.now().strftime("%Y-%m-%d")
+
+PARTITION_NAME = "{{ ds_nodash }}"
+
+INSERT_ROWS_QUERY = f"INSERT {DATASET_NAME}.{TABLE_NAME} VALUES " "(42, '{{ ds }}')"
+
+SCHEMA = [
+    {"name": "value", "type": "INTEGER", "mode": "REQUIRED"},
+    {"name": "ds", "type": "DATE", "mode": "NULLABLE"},
+]
+
+dag_id = "example_bigquery_sensors"
+
+with models.DAG(
+    dag_id,
+    schedule_interval='@once',  # Override to match your needs
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example"],
+    user_defined_macros={"DATASET": DATASET_NAME, "TABLE": TABLE_NAME},
+    default_args={"project_id": PROJECT_ID},
+) as dag_with_locations:
+    create_dataset = BigQueryCreateEmptyDatasetOperator(
+        task_id="create-dataset", dataset_id=DATASET_NAME, project_id=PROJECT_ID
+    )
+
```

Comment:
Its better to add `Start` and `End` in the docstring for every node in DAG

Reply:
Added